demo/web_api/static/static/js/main.b7f6f124.js.LICENSE.txt

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Is this license file required by some class or was it just part of a template? (In the latter case i think we can remove it)

Reply:
it was included in the build process of my React project, it included all licenses of the dependencies used in a React project. I'll check if there is any consequences or option to turn it off.